### PR TITLE
Remove Cassandra dependencies from universal/scalardl module

### PR DESCRIPTION
**Pull request by ymorimo**
https://scalar-labs.atlassian.net/browse/DLT-7405

The following works have been done to remove dependencies on Cassandra from the universal/scalardl module.
(But there is no option to switch the database other than Cassandra yet.)

* `cassandra_username` and `cassandra_password` have been renamed to `database_username` and `database_password`.
* `replication_factor` has been renamed to `cassandra_replication_factor` since it's going to be a Cassandra-specific option.
* `database_contact_points` has been added to universal/scalardl module, but `cassandra-lb` is still hardcoded in the aws/azure modules.
* `database_contact_port` is going to be needed but has not been added yet.

The base branch of this PR is `dynamodb`. I plan to open subsequent PRs toward this branch.

### CODE REVIEW

**Comment by feeblefakie** [comment]
Oh, you've done this in the story !!!
The story was supposed to be for implementing DynamoDB deployment in scalar-terraform only
and I am planning to make the ledger part cloud agnostic in https://scalar-labs.atlassian.net/browse/DLT-7496.
Anyway, it's done so it's fine. 👍

File: modules/aws/scalardl/cluster/main.tf [hunk 1]
```diff
@@ -53,10 +53,11 @@ module "scalardl_provision" {
...
-  scalardl_image_name = var.scalardl_image_name
-  scalardl_image_tag  = var.scalardl_image_tag
-  replication_factor  = var.replication_factor
-  internal_domain     = var.internal_domain
-  cassandra_username  = var.cassandra_username
-  cassandra_password  = var.cassandra_password
+  scalardl_image_name          = var.scalardl_image_name
+  scalardl_image_tag           = var.scalardl_image_tag
+  internal_domain              = var.internal_domain
+  database_contact_points      = "cassandra-lb.${var.internal_domain}" # TODO: add to variables
```
Comment (feeblefakie) [inline modules/aws/scalardl/cluster/main.tf:59]: If it's going to be done in another PR, can you create a story not to forget?

**Review by feeblefakie — APPROVED** [review]
LGTM!

**Comment by tei-k** [comment]
overall looking good. 👍 

Also need to update README.
https://github.com/scalar-labs/scalar-terraform/blob/master/modules/aws/scalardl/cluster/README.md
https://github.com/scalar-labs/scalar-terraform/blob/master/modules/azure/scalardl/cluster/README.md

**Comment by ymorimo** [comment]
@tei-k Thanks! Updated two README files.

**Review by tei-k — APPROVED** [review]
LGTM!

File: modules/aws/scalardl/cluster/main.tf (hunk 1 above)
Comment (ymorimo) [reply modules/aws/scalardl/cluster/main.tf:59]: Yes, the variable is going to be added later.
Created as a subtask https://scalar-labs.atlassian.net/browse/DLT-7605
For database_contact_port variable: https://scalar-labs.atlassian.net/browse/DLT-7524
